tests/legacy/ui/test_create_import_publish_module_and_collection.py

```diff
@@ -170,3 +170,41 @@ def test_publish_collection(self, archive_base_url, legacy_base_url,
                                         legacy_id=content_published.id).open()
         assert archive_content.title == 'CNX Automation Test Collection'
         snapshot.assert_dict_match(archive_content.stable_dict, 'legacy/col_with_m46922_1.13.json')
+
+    @markers.legacy
+    @markers.test_case('C195230')
+    @markers.slow
+    @markers.parametrize('collection_id', ['col10699'])
+    def test_reorder_author(self, legacy_base_url, legacy_username,
+                            legacy_password, collection_id, selenium):
+        # GIVEN a logged in user on their dashboard with a collection created in the previous test
+        login_page = LoginForm(selenium, legacy_base_url).open()
+        my_cnx = login_page.login(legacy_username, legacy_password)
+        collections = my_cnx.workspace_collection()
+
+        if collections.has_content is None:
+            pytest.skip('This test requires a CNX collection and '
+                        'a CNX module from previous tests that failed')
+
+        # WHEN choose the collection for test and click on Roles tab
+        cols = collections.collection_list
+
+        test_col = -1
+        for col_idx in range(len(cols)-1):
+            if cols[col_idx].collection_id == collection_id:
+                test_col = col_idx
+
+        if test_col == -1:
+            pytest.skip('This test requires a specific CNX collection '
+                        'and test failed to find')
+
+        workspace_collection_edit = cols[test_col].click_collection_link
```

Comment:
This should be a method call. you probably need to remove the `@property` decorator.

`click_collection_link()`